client/src/components/J40Header.tsx

```diff
@@ -132,7 +133,7 @@ const J40Header = ({location}:IJ40HeaderProps) => {
         </span>
         <br/>
       </Alert>
-      {!isMethodologyPage && <Alert
+      {(!isMethodologyPage && !isExploreToolPage) && <Alert
```

Comment:
Another place to avoid matching on `location` - we should look to other solutions here

Reply:
You mentioned this above:

> We should avoid relying on location in this way - components should be independent of their context of use, and the mapping from location to page is not guaranteed to be persistent / introduces a hidden dependency.
> 
> If a given page needs to do this just on the page, we should introduce a wrapper and include this as part of the styling of the page

Agree with the sentiment and would like to validate implementation:

1. Move Alerts out of J40header and wrap into `<AlertWrapper />`
3. Call `<AlertWrapper />` on each page
4. Use page styling to display or not display each Alert via `<AlertWrapper />`'s props ?

Reply:
The location has up to now been used to get flags, which are independent of the specific page they’re used on - this is a different use case than styling a component based on location — worth looking into whether there are other ways to pass the location for flag purposes, but the gatsby context by default only passes it to pages.

Reply:
just removed location from the two aforementioned places!